diff --git a/silver_platter/__init__.py b/silver_platter/__init__.py
index c1f7467..b9cda2f 100644
--- a/silver_platter/__init__.py
+++ b/silver_platter/__init__.py
@@ -29,7 +29,8 @@ if os.name == "posix":
     sys._brz_default_fs_enc = "utf8"
 
 import breezy  # noqa: E402
-breezy.initialize()
+if not breezy._global_state:
+    breezy.initialize()
 import breezy.git  # For git support   # noqa: E402
 import breezy.bzr  # For bzr support   # noqa: E402
 import breezy.plugins.launchpad  # For lp: URL support  # noqa: E402
